webserver/views/test/test_datasets.py

```diff
@@ -205,6 +211,24 @@ def test_recording_info(self):
         resp = self.client.get(url_for("datasets.recording_info", mbid=self.test_uuid))
         self.assert404(resp)
 
+    @mock.patch('webserver.external.musicbrainz.get_recording_by_id')
+    def test_recording_info_in_dataset(self, get_recording_by_id):
+        # self.test_mbid_1, is in the dataset
+        recording_mbid = "e8afe383-1478-497e-90b1-7885c7f37f6e"
```

Comment:
why not just use `self.test_mbid_1` here instead of `recording_mbid`?

Reply:
mostly because it was originally like this and I didn't change it. Good idea